CoThings/Views/Settings/QRCodeView.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  QRCodeView.swift
+//  CoThings
+//
+//  Created by Quentin on 22/05/2020.
+//  Copyright © 2020 Rainlab. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+import AVFoundation
+import UIKit
+
+struct QRCodeView: UIViewControllerRepresentable {
+    
+    private var controller: QRCodeViewController = QRCodeViewController()
```

Comment:
Do not create the view controller in here. It's life time is controlled by the swiftui framework so you should create the ui view controller whenever swiftui asks you.